### PR TITLE
Main page Pokémon Cards

### LINKED ISSUE #15: [4pt]  - Display list of items on the Homepage - Student A
Create the main part of the homepage that keeps the layout from the wireframe: 

Screenshot 2021-06-27 at 21 50 23

When the page loads, the webapp retrieves data from  the selected API and shows the list of items on screen with the basic data (e.g. title + image).

Prepare also "Comments" and "Reservations" buttons. They should be doing nothing - just being displayed.

This task **does not** include displaying the number of likes for each item.

This task **does not** include the counter of the items.

### PR DESCRIPTION
**Pull request by cancelei**
Hello Amir,

Please review this PR, and consider commenting on suggested or requested changes.

PS: I linked this PR with the Issue on our Project Dashboard, so, when this PR is closed, the issue will be resolved as well. 

Update1:
Created a JS helper file, cardUtils that has 2 functions that are imported into the index.js file, for a cleaner code base.

Regards,
Glauber

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -14,9 +16,118 @@
...
+
+        <div class="card">
+            <div class="card-img"><img class="pokeimg" id="pokeimg" alt="pokeimage"></img></div>
+            <div class="card-info">
+
+                <div class="cardtitleparent">
+                    <h2 class="cardtitle">Bulbasaur</h2>
+                    <p class="likebutton"></p>
+
+                </div>
```
Comment (Salarhabibi-amir) [inline src/index.html]: - Please make this part dynamic.

**Review by Salarhabibi-amir — CHANGES_REQUESTED** [review]
## STATUS: Required changes 🍾✔🎉
Hi @cancelei   👋 

 

## Optional suggestions
✅ You can find them in comments. And I really advise you to add them to your project as much as possible.
Thanks for sharing your work with me, Amir 🤝 

Cheers and Happy coding!👏👏👏

File: src/index.html (hunk 1 above)
Comment (cancelei) [reply src/index.html]: Thank for bringing that issue up, the instructions are not clear about that feature, but I implemented it.

**Review by Salarhabibi-amir — APPROVED** [review]
## STATUS:  Approved 🍾✔🎉
Hi @cancelei   👋 

 There is nothing to say about it.

## Optional suggestions
✅ You can find them in comments. And I really advise you to add them to your project as much as possible.
Thanks for sharing your work with me, Amir 🤝 

Cheers and Happy coding!👏👏👏

**Pull request merged**